eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Examples/Consensus.hs

```diff
@@ -135,8 +135,8 @@ defaultShelleyLedgerExamples mkWitnesses mkAlonzoTx value txBody auxData transla
     , sleHashHeader = exampleHashHeader (Proxy @era)
     , sleTx = mkAlonzoTx tx
     , sleApplyTxError =
-        ApplyTxError
-          [DelegsFailure $ DelegateeNotRegisteredDELEG @era (mkKeyHash 1)]
+        ApplyTxError . NE.singleton . DelegsFailure $
```

Comment:
Using `pure` is a neat idea. It seems like it would be a little obscure, though, since it's not obvious what it's doing. However, most of the time someone reading the code doesn't need to care. I'll think about it, and use either that or overloaded lists.